### PR TITLE
Use bblfsh argument for top_repos_quality entry point

### PR DESCRIPTION
**Pull request by zurk**
Closes https://github.com/src-d/style-analyzer/issues/513

### CODE REVIEW

File: lookout/style/format/benchmarks/top_repos_quality.py [hunk 1]
```diff
@@ -157,7 +157,7 @@ def __init__(self, quality: Optional[str] = None, model: Optional[str] = None) -
 
 
 def measure_quality(repository: str, from_commit: str, to_commit: str, port: int,
-                    review_config: dict, train_config: dict) -> QualityReport:
+                    review_config: dict, train_config: dict, bblfsh: str=None) -> QualityReport:
```
Comment (m09) [inline lookout/style/format/benchmarks/top_repos_quality.py]: Type annotation should be `Optional[str]` and we should probably remove the default value to make sure no part of the codebase is mistakingly forgetting to pass on bblfsh arg (now or in the future).

File: lookout/style/format/benchmarks/top_repos_quality.py [hunk 2]
```diff
@@ -376,7 +380,7 @@ def create_parser() -> ArgumentParser:
         help="If this flag is used - force to overwrite results stored in output directory. "
              "If not - stored results will be used if they exist.")
     parser.add_argument(
-        "-b", "--bblfsh", default="localhost:9432", help="Bblfsh address to use.")
+        "-b", "--bblfsh", default=None, help="Bblfsh address to use.")
```
Comment (m09) [inline lookout/style/format/benchmarks/top_repos_quality.py]: `default=None` can be removed

File: lookout/style/format/benchmarks/top_repos_quality.py [hunk 3]
```diff
@@ -157,7 +157,7 @@ def __init__(self, quality: Optional[str] = None, model: Optional[str] = None) -
 
 
 def measure_quality(repository: str, from_commit: str, to_commit: str, port: int,
-                    review_config: dict, train_config: dict) -> QualityReport:
+                    review_config: dict, train_config: dict, bblfsh: str) -> QualityReport:
```
Comment (m09) [inline lookout/style/format/benchmarks/top_repos_quality.py]: Type annotation should still be Optional[str]

File: lookout/style/format/benchmarks/top_repos_quality.py [hunk 4]
```diff
@@ -167,6 +167,7 @@ def measure_quality(repository: str, from_commit: str, to_commit: str, port: int
     :param port: Port for QualityReportAnalyzer.
     :param review_config: config for review.
     :param train_config: config for train.
+    :param bblfsh: Babelfish server address to use. Specify None to use default value.
```
Comment (vmarkovtsev) [inline lookout/style/format/benchmarks/top_repos_quality.py]:
```suggestion
    :param bblfsh: Babelfish server address to use. Specify None to use the default value.
```

File: lookout/style/format/benchmarks/top_repos_quality.py [hunk 5]
```diff
@@ -376,7 +380,7 @@ def create_parser() -> ArgumentParser:
         help="If this flag is used - force to overwrite results stored in output directory. "
              "If not - stored results will be used if they exist.")
     parser.add_argument(
-        "-b", "--bblfsh", default="localhost:9432", help="Bblfsh address to use.")
+        "-b", "--bblfsh", help="Bblfsh address to use.")
```
Comment (vmarkovtsev) [inline lookout/style/format/benchmarks/top_repos_quality.py]: Why do we remove the default value? It is a nice hint in `--help`...

File: lookout/style/format/benchmarks/top_repos_quality.py (hunk 3 above)
Comment (zurk) [reply lookout/style/format/benchmarks/top_repos_quality.py]: oops, redundant space...

File: lookout/style/format/benchmarks/top_repos_quality.py (hunk 5 above)
Comment (vmarkovtsev) [reply lookout/style/format/benchmarks/top_repos_quality.py]: What if `LOOKOUT_BBLFSHD ` is not defined? Will it crash?

**Comment by vmarkovtsev** [comment]
@m09 all good?

**Comment by m09** [comment]
wow you clicked the merge button faster than me and I was already on the page :D

**Pull request merged**